### ISSUE TITLE
JupyterLab version (v3 to v4) update

### ISSUE THREAD

**Issue by ftorradeflot** [rucio/jupyterlab-extension#27]

**Comment by ftorradeflot** [comment]
There's an extension migration guide: https://jupyterlab.readthedocs.io/en/4.0.x/extension/extension_migration.html

**Comment by garciagenrique** [comment]
Summary of the `jupyter/base-notebook` - `JupyterLab` version relationship on the [repo wiki](https://github.com/rucio/jupyterlab-extension/wiki).

@ftorradeflot If I recall correctly, did you mention that the python version of the `jupyter/scipy-notebook` that you were spawning were coming with jlab v3 and python >3.9 ? I would like to know which is this `scipy-notebook` version and where are you pulling it from. Thanks !

**Comment by ftorradeflot** [comment]
No, I couldn't find a version with python 3.9. I created my own environment using another image. The scripts to launch a test environment are almost ready, see my fork of the project

**Comment by garciagenrique** [comment]
Current progress on fork: 
https://github.com/garciagenrique/jupyterlab-extension/pull/5
